src/main/scala/com/codacy/parsers/implementation/CloverParser.scala

```diff
@@ -43,9 +44,23 @@ object CloverParser extends CoverageParser with XmlReportParser {
     MathUtils.computePercentage(coveredStatements, totalStatements)
   }
 
-  private def getCoverageFileReport(rootPath: String, fileNode: Node) = {
-    val filename = fileNode \@ "name"
-    val cleanFileName = TextUtils.sanitiseFilename(filename).stripPrefix(rootPath).stripPrefix("/")
+  private def getCoverageFileReport(rootPath: String, fileNode: Node): CoverageFileReport = {
+    val filePath = fileNode.attribute("path").flatMap(_.headOption.map(_.text)).map(TextUtils.sanitiseFilename)
+    val filename = fileNode.attribute("name").flatMap(_.headOption.map(_.text)).map(TextUtils.sanitiseFilename)
+
+    val relativeFilePath = filePath
```

Comment:
I could not find a spec for the format so I honestly cannot understand if this is the best solution :cry: